OpenGL/src/Renderer.cpp

```diff
@@ -42,16 +43,29 @@ void Renderer::Draw(const VertexArray& va, const IndexBuffer& ib, const Shader&
    GLCall(glDrawElements(GL_TRIANGLES, ib.GetCount(), GL_UNSIGNED_INT, nullptr));
 }
 
-void Renderer::Line(glm::vec2 start, glm::vec2 end, glm::vec3 color) {
+void Renderer::DrawLine(glm::vec2 start, glm::vec2 end, glm::vec4 color) {
    lineShader.Bind();
-   lineShader.SetUniform4f("u_Color", glm::vec4(color.r, color.g, color.b, 0.3f));
-   lineShader.SetUniform2f("u_StartPos", glm::vec2(start.x, start.y) - Camera::position + glm::vec2{9.0, 9.0});
-   lineShader.SetUniform2f("u_EndPos", glm::vec2(end.x, end.y) - Camera::position + glm::vec2{9.0, 9.0});
+   lineShader.SetUniform4f("u_Color", color);
+   lineShader.SetUniform2f("u_StartPos", glm::vec2(start.x, start.y));
+   lineShader.SetUniform2f("u_EndPos", glm::vec2(end.x, end.y));
```

Comment:
I believe you can use:

```suggestion
   lineShader.SetUniform2f("u_EndPos", glm::vec2(end));
```

Reply:
Done